pureport/api/client.py

```diff
@@ -188,6 +190,13 @@ def login(self, key=None, secret=None, access_token=None, profile=None, api_url=
                                         file_credentials['api_secret'])
         raise MissingAccessTokenException()
 
+    def open_api(self):
+        """
+        Get's the open api documentation
+        :rtype: dict
+        """
+        return self.__session.get('/openapi.json').json()
```

Comment:
We may want to allow specifying the `openapi.json` file path via constructor.

Reply:
It's not going to really change though, right?

Reply:
I'm thinking that we may want to test it against multiple versions API.
But we can update that later I suppose.